static/src/stylesheets/layout/nav/_top-bar-dropdown.scss

```diff
@@ -90,7 +90,7 @@ from scrolling */
     display: block;
     letter-spacing: .01rem;
     outline: none;
-    padding: ($gs-baseline / 2 + 1) $gs-gutter ($gs-baseline + $gs-baseline / 4) ($gs-gutter / 2);
+    padding: ($gs-baseline / 2 + 1) $gs-gutter ($gs-baseline + $gs-baseline / 4);
```

Comment:
the frontend menu differs from the dcr menu in that there is very little left padding on the items, which means there's no space for the red dot. This line changes that